Pod/Classes/PhxPush.h

```diff
@@ -9,17 +9,21 @@
 #import <Foundation/Foundation.h>
 #import "PhxTypes.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @class PhxChannel;
 
 @interface PhxPush : NSObject
 
-- (id)initWithChannel:(PhxChannel*)channel
-                event:(NSString*)event
-              payload:(NSDictionary*)payload;
+- (instancetype)initWithChannel:(PhxChannel*)channel
+                          event:(NSString*)event
+                        payload:(NSDictionary*)payload;
 
 - (void)send;
 
-- (PhxPush*)onReceive:(NSString*)status callback:(OnMessage)callback;
-- (PhxPush*)after:(int)ms callback:(After)callback;
+- (PhxPush *)onReceive:(NSString *)status callback:(OnMessage)callback;
+- (PhxPush *)after:(NSTimeInterval)ms callback:(After)callback;
```

Comment:
this is not actually a correct change *if the original parameter was in milliseconds*. The parameter sounds like it's in milliseconds (`ms`), if that's the case `NSTimeInterval` is different because it's base-size is seconds. 

I'm all for using `NSTimeInterval` but if you do, you have to convert the value from milliseconds to seconds.